javascript/008-mixed-juices/mixed-juices.spec.js

```diff
@@ -0,0 +1,113 @@
+import { timeToMixJuice, limesToCut, remainingOrders } from './mixed-juices';
+
+describe('timeToMixJuice', () => {
+  test("returns the correct time for 'Pure Strawberry Joy'", () => {
+    expect(timeToMixJuice('Pure Strawberry Joy')).toBe(0.5);
+  });
+
+  test('returns the correct times for all other standard menu items', () => {
+    expect(timeToMixJuice('Energizer')).toBe(1.5);
+    expect(timeToMixJuice('Green Garden')).toBe(1.5);
+    expect(timeToMixJuice('Tropical Island')).toBe(3);
+    expect(timeToMixJuice('All or Nothing')).toBe(5);
+  });
+
+  test('returns the same time for all other juices', () => {
+    const defaultTime = 2.5;
+    expect(timeToMixJuice('Limetime')).toBe(defaultTime);
+    expect(timeToMixJuice('Manic Organic')).toBe(defaultTime);
+    expect(timeToMixJuice('Papaya & Peach')).toBe(defaultTime);
+  });
+});
+
+describe('limesToCut', () => {
+  test('calculates the number of limes needed to reach the target supply', () => {
+    const limes = [
+      'small',
+      'large',
+      'large',
+      'medium',
+      'small',
+      'large',
+      'large',
+      'medium',
+    ];
+    expect(limesToCut(42, limes)).toBe(6);
+
+    expect(limesToCut(4, ['medium', 'small'])).toBe(1);
```

Comment:
Add a test when number of lime wedges we need to cut is exactly satisfied by taking a few limes from an array.

Reply:
This still stands